pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -388,6 +388,16 @@ func (c *HAProxy) Volumes(podSpec *api.PodSpec, cr *api.PerconaXtraDBCluster, vg
 		vol.Volumes = append(vol.Volumes,
 			app.GetConfigVolumes("hookscript", c.labels["app.kubernetes.io/instance"]+"-"+c.labels["app.kubernetes.io/component"]+"-hookscript"))
 	}
+	if cr.CompareVersionWith("1.13.0") >= 0 {
+		vol.Volumes = append(vol.Volumes,
+			corev1.Volume{
+				Name: app.DataVolumeName,
```

Comment:
I'm not sure about adding an `emptyDir` volume as datadir. Can we change the name?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1417/commits/2de93aad00deb6cfe38b7e17dcf73ec86150ae13